spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -211,6 +211,7 @@ object SpdxDeclaredLicenseMapping {
         "cc by-sa 4.0" to CC_BY_SA_4_0.toExpression(),
         "cpal 1.0" to CPAL_1_0.toExpression(),
         "cpal v1.0" to CPAL_1_0.toExpression(),
+        "DBAD" to licenseRef("dbad", "ort"),
```

Comment:
Just FYI, I've checked and this license is available in ScanCode starting with version 3.1.0, but as we're stuck with ScanCode 3.0.2 it's correct to use the "ort" namespace instead of "scancode".

Reply:
@sschuberth  Shall I align the with ScanCode to include the version number e.g. change `licenseRef("dbad", "ort")` to `licenseRef("dbad-1.1", "ort")`?

Reply:
No, I believe this is fine as-is. We probably need to / should clean up the `LicenseRefs` here anyway once we upgrade ScanCode.

*Or* we should actually go exactly the other way around and not emit *any* scanner-specific `LicenseRef`s here, but only "ort" `LicenseRef`s. Even if that means that we might need to map some "ort" and "scancode" `LicenseRef`s to the same `LicenseRef` so they are basically treated as aliases in the rules. But doing something scanner-specific here generally feels wrong to me.

Reply:
I think it would be a good idea for ort to only produce LicenseRef-ort but we have to provide ORT users a way to remap LicenseRef-ort to LicenseRef-scancode or LicenseRef-MyCompanyName identifiers.

Following up our discussion in the last TSC call we should move from a hard-coded list to a   `license-mapping.yml` so the user can define their own mappings. Not yet sure whether we should still have a hard-coded list in ORT as a default/fallback.

Reply:
Regarding the hard-coded list as default/fallback you mention, we can put the defaults into `license-mapping.yml` file rather than compiling it in.